http.go

```diff
@@ -715,6 +715,18 @@ func (wc *WalletClient) UpsertContactForPaymail(ctx context.Context, paymail, fu
 	return &result, nil
 }
 
+// GetSharedConfig gets the shared config
+func (wc *WalletClient) GetSharedConfig(ctx context.Context) (*models.SharedConfig, ResponseError) {
+	var model *models.SharedConfig
+	if err := wc.doHTTPRequest(
+		ctx, http.MethodGet, "/shared-config", nil, wc.xPriv, true, &model,
+	); err != nil {
+		return nil, err
+	}
```

Comment:
That would not work with our "standard flow" with `web-backend`, because you hardcoded the `xPriv` 
- but there is also the `adminXPriv` - and the `web-backend` is using the adminKey to access the shared config.

I'm proposing sth like this - I've tested it with `web-backend` and `web-frontend` (with your changes in `spv-wallet`) 
and it worked 😄 
```suggestion
	key := wc.xPriv
	if wc.adminXPriv != nil {
		key = wc.adminXPriv
	}
	if key == nil {
		return nil, WrapError(errors.New("neither xPriv nor adminXPriv is provided"))
	}
	if err := wc.doHTTPRequest(
		ctx, http.MethodGet, "/shared-config", nil, key, true, &model,
	); err != nil {
		return nil, err
	}
```

Reply:
Changed